src/OmniSharp/Models/v1/HighlightResponse.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Text;
+
+namespace OmniSharp.Models
+{
+    public class HighlightResponse : IComparable<HighlightResponse>
+    {
+        public int StartLine { get; set; }
+        public int StartColumn { get; set; }
+        public int EndLine { get; set; }
+        public int EndColumn { get; set; }
+        public string Kind { get; set; }
+        public IEnumerable<string> Projects { get; set; }
+
+        internal static HighlightResponse FromClassifiedSpan(ClassifiedSpan span, TextLineCollection lines, IEnumerable<string> projects)
+        {
+            var linePos = lines.GetLinePositionSpan(span.TextSpan);
+
+            return new HighlightResponse
+            {
+                StartLine = linePos.Start.Line + 1,
+                EndLine = linePos.End.Line + 1,
+                // Caution, Character is 1-based.
+                StartColumn = linePos.Start.Character,
+                EndColumn = linePos.End.Character,
+                Kind = span.ClassificationType,
+                Projects = projects
+            };
+        }
+
+        public int CompareTo(HighlightResponse other)
```

Comment:
return other.StartLine.CompareTo(StartLine) * other.StartColumn.CompareTo(StartColumn) ? not 100% but seem like it would do the same thing as this method